spec/per_denomination_strategy_spec.rb

```diff
@@ -0,0 +1,106 @@
+require 'spec_helper'
+
+describe Trebuchet::Strategy::PerDenomination do
```

Comment:
used `.should` style syntax for internal consistency with other strategy specs. open to switching to `expect`-style syntax if desirable